libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -2024,6 +2076,60 @@ bool WaveTrack::Get(samplePtr buffer, sampleFormat format,
    return result;
 }
 
+std::vector<LibSampleTrack::TrackBlockSequence> WaveTrack::GrabFloats(
+   size_t iChannel, size_t nBuffers, double& t0, sampleCount length) const
+{
+   const auto nChannels = NChannels();
+   assert(nBuffers - iChannel <= nChannels); // precondition
```

Comment:
If you say it's a precondition, then int should be commented where the member function is declared in the header file, preferably with the `@pre` Doxygen comment convention